pages/integrations/sso/azure_ad.md.erb

```diff
@@ -0,0 +1,111 @@
+1. Click 'Add user/group'
+1. Select the user and click 'Assign'
+
+Then, retry the test login from to SSO provider page on Buildkite.
+
+## Step 4. Enable the new SSO Provider
+
+Once you've performed a test login you can enable your provider using the `Enable` button. Activating SSO will not force a log out of existing users, but will cause all new or expired sessions to authorize through Cloud Identity before organization data can be accessed.
+
+If you need to edit or update your Azure Active Directory provider settings at any time, you will need to disable the provider first. For more information on disabling a provider, see the [disabling SSO](/docs/integrations/sso#disabling-and-removing-sso) section of the SSO overview.
+
+## Using SCIM to provision and manage users
+
+<div class="Docs__note">
+  <p>This integration with AzureAD is currently available only to a limited number of customers. Contact support@buildkite.com to learn more.</p>
+</div>
+
+### Supported SCIM Features
+
+* Create Users (note that users added to Okta won't be billed for until they have signed up with Buildkite)
+* Deactivate Users (Deprovisioning)
+
+### Configuration instructions
+
+Using the SCIM provisioning settings in AzureAD, Enterprise customers can automatically remove user accounts from your Buildkite organization. In AzureAD this feature is called 'Provisioning'. You need an enabled AzureAD SSO Provider before you can set up SCIM.
+
+After creating your SSO Provider in Buildkite, you will need the  _Base URL_ and _API Token_ from your AzureAD SSO Provider Settings:
+
+<%= image "azuread-scim-settings.png", width: 1440/2, height: 548/2, alt: "Screenshot of the Buildkite Settings SCIM Deprovisioning section" %>
+
+Go to your Buildkite application in AzureAD to set up provisioning:
+
+1. Return to the Azure Admin Console and select the new 'Buildkite' enterprise app
+1. Choose 'Provisioning' from the navigation sidebar
+1. Click 'Get started'
+1. Select 'Automatic' provisioning mode and enter the following details:
+    * Tenant URL: the Base URL from your Buildkite SSO Provider settings
+    * Secret Token: the API Token from your Buildkite SSO Provider settings
+1. Click 'Test Connection'
+1. Click 'Save' once you receive confirmation the settings are valid
+1. Disable Group synchronization:
+    * Expand 'Mappings'
+    * Click 'Provision Azure Active Directory Groups'
+    * Toggle 'Enabled' to 'No' and click 'Save'
+    * Return to provisioning setttings
+1. Customise the User mappings
+    * Expand 'Mappings'
+    * Click 'Provision Azure Active Directory Users'
+    * Click the first row of the 'Attribute Mappings' table ('userPrincipalName'), change the 'Source attribute' to 'mail' and click 'OK'
+    * Delete all remaining mappings except for the four listed in the following screenshot:
+    <%= image "azuread-user-mappings.png", width: 1440/2, height: 548/2, alt: "Screenshot of the required user mappings on Azure" %>
```

Comment:
What I don't know is where this is coming from, which looks like the odd spacing?

```
<div style="max-width: 720px"><div class="responsive-image-container" style="padding-bottom: 38.05555555555556%">
```